x/ecocredit/client/tx.go

```diff
@@ -52,14 +57,139 @@ func txCreateClass() *cobra.Command {
 				return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "base64_metadata is malformed, proper base64 string is required")
```

Comment:
```suggestion
				return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "metadata is malformed, proper base64 string is required")
```

Note to update this if we indeed end up going with "metadata" instead of "base64_metdata" for the param name. (same with line 53).